.env.example

```
@@ -9,9 +9,9 @@
 # When adding additional environment variables, the schema in "/src/env.js"
 # should be updated accordingly.
 
+# Better Auth
+BETTER_AUTH_SECRET="secret"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)